index.js

```diff
@@ -87,11 +103,11 @@ export function createStartPage (categoriesData) {
 	productslist.createProductsList ();
 }
 
-function enableButton (categoriesData) {
+function enableButton () {
```

Comment:
What button?
(Yes, I know I can see a line below that it's the "Add" button, but let's get into some good habits. ;))